### DISCUSSION TITLE
non-ACSII text in wiresl2 comment

### DISCUSSION THREAD

**Discussion by caryr** [steveicarus/iverilog#471] [General]
The standard specifically requires that the comment text be ASCII (in the appendix A BNF). In the wiresl2 test there is some non-ASCII text that is being caught by a tool that is strictly following the standard. Do we want to change the comment to remove the non-ASCII text or should I just exclude this from what the other tool is using with a comment regarding the exclusion?

I'm not sure what the commercial tools support, but I do understand this could be beneficial to non-English speakers/writers. I will check the commercial tool I use, but it would be nice to know what others do. I assume if the commercial tools support this we should also. Thoughts?

**Comment by caryr** [comment]
I do not have a good setup for this since my work machine does not appear to display non-ASCII characters, but if I trust the non-displaying characters I carefully typed it seems like at least one commercial tools allow non-ASCII characters in comments. It would be nice for someone who uses non-ASCII regularly and has access to commercial tools to test this.

**Comment by caryr** [comment]
I figured out how to get the actual file on my work machine without violating any of the restrictions we have and it ran correctly. The comment did look different so hopefully there wasn't any funky translation going on.

**Comment by martinwhitaker** [comment]
I think we should continue to allow non-ASCII characters in comments, regardless of what other simulators do. But you could change the comment in wiresl2.v to just say that vvp output random bytes following the correct text output - it's really not important to know what those bytes were on one particular run.

**Comment by caryr** [comment]
Yes, that was my thought.

**Comment by caryr** [comment]
Closing this discussion since it is complete.
